core/src/main/scala/kafka/tools/PerfConfig.scala

```diff
@@ -40,20 +40,5 @@ class PerfConfig(args: Array[String]) {
   val showDetailedStatsOpt = parser.accepts("show-detailed-stats", "If set, stats are reported for each reporting " +
```

Comment:
Thanks for the comments. Seems it is also used by `SimpleConsumerPerformance`, and both `ConsumerPerformance` and `SimpleConsumerPerformance` defines their own `ConsumerPerfConfig` class.